app/assets/javascripts/discourse/routes/review-index.js.es6

```diff
@@ -30,6 +31,24 @@ export default DiscourseRoute.extend({
     });
   },
 
+  activate() {
+    this.messageBus.subscribe("/reviewable_claimed", data => {
+      const reviewables = this.controller.reviewables;
+      if (reviewables) {
+        const user = data.user ? User.create(data.user) : null;
```

Comment:
This will skip the identity map in the store that I suspect is already used for `claimed_by`. Could you use a `store` method to rehyrdrate the user?

Also, does this work when it's unclaimed, by setting `claimed_by` to null?

Reply:
Thanks for the `store` suggestion, that is really the best way.

Yes, it works when `claimed_by` is set to null.